src/main/resources/db/migration/data/dev/R__1_1__locations.sql

```diff
@@ -1,2 +1,8 @@
-insert into locations (location_id, added_at, location_type, nomis_agency_id, site_name) select * from (select uuid('709fbee3-7fe6-4584-a8dc-f12481165bfa'), current_timestamp, 'PR', 'GNI', 'HMP Grendon') as tmp where not exists (select location_id from locations where location_id = '709fbee3-7fe6-4584-a8dc-f12481165bfa');
```

Comment:
I don't want these mapped anymore we need to allow for mapping via the call to BaSM and these are genuine location agency id's.